### PR TITLE
Update info regarding authenticator option changes.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: en/appendices/3-1-migration-guide.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ Controller
 
     // In a controller, instead of
     $this->layout = ('advanced');
```
Comment (HavokInspiration) [inline en/appendices/3-1-migration-guide.rst]: Not related to your changes, but it should be ``$this->layout = 'advanced';``

File: en/controllers/components/authentication.rst [hunk 2]
```diff
@@ -105,11 +105,10 @@ keys.
   ``username`` and ``password`` to specify your username and password fields
   respectively.
 - ``userModel`` The model name of the users table; defaults to Users.
-- ``scope`` Additional conditions to use when looking up and
-  authenticating users, i.e. ``['Users.is_active' => true]``.
-- ``contain`` Extra models to contain and return with identified user's info.
+- `finder` The finder method to use to fetch user record. Defaults to 'all'.
```
Comment (ravage84) [inline en/controllers/components/authentication.rst]: > user record

Shouldn't this be either "user record**s**" or "**a** user record"?

Comment (ADmad) [inline en/controllers/components/authentication.rst]: Fell free to improve language / grammar as you see fit.

Comment (ravage84) [inline en/controllers/components/authentication.rst]: I already did, but in this case, I'm not sure.

**Comment by markstory** [comment]
I did my suggestions as a commit.

**Pull request merged**